packages/driver/src/cypress/log.js

```diff
@@ -153,9 +153,17 @@ const defaults = function (state, config, obj) {
       },
     })
 
-    // if obj.isCurrent
-    // stringify the obj.message (if it exists) or current.get("args")
-    obj.message = $utils.stringify(obj.message != null ? obj.message : (current != null ? current.get('args') : undefined))
+    let args = current != null ? current.get('args') : undefined
+
+    if (args) {
+      const last = _.last(args)
+
+      if (_.isObject(last)) {
+        args = args.slice(0, args.length - 1)
+      }
+    }
```

Comment:
see prev comment, we can't assume last arg is always `options`

Reply:
How about implementing `getIndexOfUserOptions(name, args)` and rewrite code like below:

```js
let args = current != null ? current.get('args') : undefined

if (args) {
  const optionsIndex = getIndexOfUserOptions(current.get('name'), args)

  if (optionsIndex !== -1) {
    args = _.remove(args, (v, i) => {
      return i === optionsIndex
    })
  }
}
```

Or is there a better solution? 

Maybe we need to implement `getUserOptions`, `getIndexOfUserOptions` first. Then, the better ideas will come. 

What do you think?

Reply:
I went through [a to l in the list](https://docs.cypress.io/api/api/table-of-contents.html) and found 5 commands that breaks this assumption. 

I'll look through other commands later and I guess this fix will be done before next Wednesday (Feb 5th)